tests/installation/grub_test.pm

```diff
@@ -69,19 +69,33 @@ sub run {
     # 60 due to rare slowness e.g. multipath poo#11908
     # 90 as a workaround due to the qemu backend fallout
     # If grub timeout was not disabled, we wait for linux-login instead
-    my $tag = get_var('KEEP_GRUB_TIMEOUT') ? 'linux-login' : 'grub2';
-    assert_screen_with_soft_timeout($tag, timeout => 2 * $timeout, soft_timeout => $timeout, bugref => 'boo#1120256');
-    stop_grub_timeout;
-    boot_into_snapshot if get_var("BOOT_TO_SNAPSHOT");
-    send_key_until_needlematch("bootmenu-xen-kernel", 'down', 10, 5) if get_var('XEN');
-    if ((check_var('ARCH', 'aarch64') && is_sle && get_var('PLYMOUTH_DEBUG'))
-        || get_var('GRUB_KERNEL_OPTION_APPEND'))
+    #my $tag = get_var('KEEP_GRUB_TIMEOUT') ? 'linux-login' : 'grub2';
+
+    #assert_screen_with_soft_timeout($tag, timeout => 2 * $timeout, soft_timeout => $timeout, bugref => 'boo#1120256');
+
+    check_screen ['grub2', 'displaymanager', 'linux-login'], 120;
+
+    if (match_has_tag('displaymanager') or match_has_tag('linux-login')) {
+        record_soft_failure("System already booted poo#49751", "Grub2 not found, because system has already booted");
```

Comment:
record_soft_failure only accepts one parameter. A verification run would show you that.

```suggestion
        record_soft_failure("poo#49751 - Grub2 not found, because system has already booted");
```